test/parallel/test-assert-calltracker-verify.js

```diff
@@ -6,27 +6,42 @@ const assert = require('assert');
 
 const tracker = new assert.CallTracker();
 
-const msg = 'Function(s) were not called the expected number of times';
+const generic_msg = 'Functions were not called the expected number of times';
 
 function foo() {}
 
+function bar() {}
+
 const callsfoo = tracker.calls(foo, 1);
+const callsbar = tracker.calls(bar, 1);
 
-// Expects an error as callsfoo() was called less than one time.
+// Expects an error as callsfoo() and callsbar() were called less than one time.
 assert.throws(
   () => tracker.verify(),
-  { message: msg }
+  { message: generic_msg }
 );
 
 callsfoo();
 
-// Will throw an error if callsfoo() isn't called exactly once.
-tracker.verify();
```

Comment:
We don't want to remove this, we want to be sure that the API doesn't throw when it should not (we can move it around, or create a separate tracker).